engine/input/src/input.cpp

```diff
@@ -356,6 +373,8 @@ namespace dmInput
 
     void DeleteBinding(HBinding binding)
     {
+        binding->m_Context->m_Binding = 0;
```

Comment:
Since we only ever use one Binding, I see no big issue with us doing this connection.